NeonArenaMvp/Game/Behaviors/Tile/TileMoveBehaviors.cs

```diff
@@ -6,16 +6,15 @@ public static class TileMoveBehaviors
     {
         public delegate MoveAction? TileMoveBehavior(MoveAction currentMoveAction);
 
-        public static readonly TileMoveBehavior PassThrough = (currentMoveAction) =>
-        {
-            return new MoveAction
-            (
-                coords: currentMoveAction.Coords.NextInDirection(currentMoveAction.Direction),
-                direction: currentMoveAction.Direction,
-                remainingRange: currentMoveAction.RemainingRange - 1,
-                previousCoords: currentMoveAction.Coords,
-                playerId: 1
-            );
-        };
+        public static readonly TileMoveBehavior PassThrough = (currentMoveAction) => new MoveAction
+        (
+            coords: currentMoveAction.Coords.NextInDirection(currentMoveAction.Direction),
+            direction: currentMoveAction.Direction,
+            remainingRange: currentMoveAction.RemainingRange - 1,
+            previousCoords: currentMoveAction.Coords,
+            playerId: currentMoveAction.PlayerId
+        );
+
+        public static readonly TileMoveBehavior Blocked = (currentMoveAction) => null;
```

Comment:
mislam deka e bolje da gi imenuvame behaviorite kako akcii, taka sto ova podobro e Block da se vika; eksplicitno kazuva sto pravi behaviorot

```suggestion
        public static readonly TileMoveBehavior Block = (currentMoveAction) => null;
```